contracts/chain-adapters/Blast_Adapter.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import "./interfaces/AdapterInterface.sol";
+import "../external/interfaces/WETH9Interface.sol";
+
+// @dev Use local modified CrossDomainEnabled contract instead of one exported by eth-optimism because we need
+// this contract's state variables to be `immutable` because of the delegateCall call.
+import "./CrossDomainEnabled.sol";
+import "@eth-optimism/contracts/L1/messaging/IL1StandardBridge.sol";
+import "@eth-optimism/contracts/L1/messaging/IL1ERC20Bridge.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "../libraries/CircleCCTPAdapter.sol";
+import "../external/interfaces/CCTPInterfaces.sol";
+
+/**
+ * @notice Contract containing logic to send messages from L1 to Blast. This is a modified version of the Optimism adapter
+ * that excludes the custom bridging logic. It differs from the Base Adapter in that it uses a special
+ * Blast contract to bridge WETH and DAI, which are yielding rebasing tokens on L2, WETH and USDB.
+ */
+
+// solhint-disable-next-line contract-name-camelcase
+contract Blast_Adapter is CrossDomainEnabled, AdapterInterface, CircleCCTPAdapter {
+    using SafeERC20 for IERC20;
+    uint32 public constant L2_GAS_LIMIT = 200_000;
```

Comment:
nit: should we make this immutable and pass it in in the constructor, so it can be changed without changing the contract code if we wanted to redeploy with a different value?